### ISSUE TITLE
Filter by facets on a subset of collections

### ISSUE THREAD

**Issue by apferrar** [pulibrary/pulfalight#153]
Should be able to filter by a few different selections (i.e. search "ACLU" and be able to choose all relevant collections). Creating mini-sets of collections to run searches against.

**Comment by tpendragon** [comment]
This is really complicated and would require a new faceting search UI that implies "OR" instead of "AND," I think. What are the chances we can push this post-MVP @ccleeton @faithc?

**Comment by faithc** [comment]
I think this could be part of 1.x. @ccleeton?

**Comment by ccleeton** [comment]
Agreed, as long as the top level restriction will be pulled in as it had previously, which I think if I understand what Faith told me was going to happen.

**Comment by tpendragon** [comment]
Thanks!
